src/js/data/CoinInfo.js

```diff
@@ -110,53 +109,6 @@ export const fixCoinInfoNetwork = (ci: BitcoinNetworkInfo, path: number[]) => {
     return coinInfo;
 };
 
-const detectBtcVersion = data => {
-    if (data.subversion == null) {
-        return 'btc';
-    }
-    if (data.subversion.startsWith('/Bitcoin ABC')) {
-        return 'bch';
-    }
-    if (data.subversion.startsWith('/Bitcoin Cash')) {
-        return 'bch';
-    }
-    if (data.subversion.startsWith('/Bitcoin Gold')) {
-        return 'btg';
-    }
-    return 'btc';
-};
-
-export const getCoinInfoByHash = (hash: string, networkInfo: any) => {
```

Comment:
i guess this should be implemented instead of removing it, afaik this was designed to validate backend connection by 1st blockhash (data source from trezor-common)
it should be used every time you connect to a backend which may be malicious, it may claim that the shortcut is "Testnet" but in fact it will be "Btc"